uns_plugin/schema_validation/validator_benchmark_test.go

```
@@ -0,0 +1,410 @@
+						"additionalProperties": false
+					}
+				},
+				"additionalProperties": false
+			}
+		},
+		"required": ["virtual_path", "fields"],
+		"additionalProperties": false
+	}`)
+
+	// Load the schema
+	validator.LoadSchema("_sensor_data", 1, sensorSchema)
```

golangci-lint (1.64.8)

Error return value of `validator.LoadSchema` is not checked

(errcheck)